diff --git a/node_modules/bob-the-bundler/dist/commands/build.js b/node_modules/bob-the-bundler/dist/commands/build.js
index 6702319..e7b623f 100644
--- a/node_modules/bob-the-bundler/dist/commands/build.js
+++ b/node_modules/bob-the-bundler/dist/commands/build.js
@@ -53,13 +53,17 @@ function assertTypeScriptBuildResult(result) {
 async function buildTypeScript(buildPath, options = {}) {
     assertTypeScriptBuildResult(await execa('npx', [
         'tsc',
+        '--project',
+        'tsconfig.build.json',
         ...compilerOptionsToArgs(typeScriptCompilerOptions('esm')),
         ...(options.incremental ? ['--incremental'] : []),
         '--outDir',
-        join(buildPath, 'esm'),
+        join(buildPath, 'esm')
     ]));
     assertTypeScriptBuildResult(await execa('npx', [
         'tsc',
+        '--project',
+        'tsconfig.build.json',
         ...compilerOptionsToArgs(typeScriptCompilerOptions('cjs')),
         ...(options.incremental ? ['--incremental'] : []),
         '--outDir',
@@ -118,7 +122,7 @@ export const buildCommand = createCommand(api => {
             }
             await buildTypeScript(bobBuildPath, { incremental });
             await Promise.all(packageInfoList.map(({ cwd, pkg, fullName }) => limit(async () => {
-                const getBuildPath = (target) => join(cwd.replace('packages', join('.bob', target)), 'src');
+                const getBuildPath = (target) => join(cwd.replace('packages', join('.bob', target, 'packages')), 'src');
                 const distPath = join(cwd, 'dist');
                 await build({
                     cwd,
