diff --git a/node_modules/bob-the-bundler/dist/commands/build.js b/node_modules/bob-the-bundler/dist/commands/build.js
index e89c398..a836601 100644
--- a/node_modules/bob-the-bundler/dist/commands/build.js
+++ b/node_modules/bob-the-bundler/dist/commands/build.js
@@ -128,7 +128,7 @@ export const buildCommand = createCommand(api => {
             }
             await buildTypeScript(bobBuildPath, { cwd, tsconfig, incremental });
             await Promise.all(packageInfoList.map(({ cwd, pkg, fullName }) => limit(async () => {
-                const getBuildPath = (target) => join(cwd.replace('packages', join('.bob', target)), 'src');
+                const getBuildPath = (target) => join(process.cwd(), '.bob', target);
                 const distPath = join(cwd, 'dist');
                 await build({
                     cwd,
